content/posts/weekly/January-2023.md

```diff
@@ -31,3 +31,9 @@ weight: 54
 - When configuring Copr chroot (target in Packit terminology) specific configuration, make sure to specify additional_modules as a string: module names separated with a comma, example: "httpd:2.4,python:4". ([packit#1826](https://github.com/packit/packit/pull/1826))
 - We have fixed a bug which caused long Copr build end reporting time on a few occurrences. ([packit-service#1838](https://github.com/packit/packit-service/pull/1838))
 - A few fixes has been implemented to make it possible to use [`propose-downstream` job](https://packit.dev/docs/configuration/#propose_downstream) on GitLab. ([packit-service#1842](https://github.com/packit/packit-service/pull/1842), [packit-service#1844](https://github.com/packit/packit-service/pull/1844), [packit-service#1845](https://github.com/packit/packit-service/pull/1845), [packit-service#1846](https://github.com/packit/packit-service/pull/1846))
+
+## Week 3 (January 17th – January 23rd)
+
+- Users can now re-trigger a _Bodhi update_ or a _Koji build_ writing comments like `/packit create-update` and `/packit koji-update` in the issue opened by Packit in the issues repository if anything went wrong during the _propose-downstream_ ([packit-service#1796](https://github.com/packit/packit-service/pull/1796))
+- Update React to v18 and related packages ([dashboard#203](https://github.com/packit/dashboard/pull/203))
```

Comment:
```suggestion
- React and related packages were updated to v18  ([dashboard#203](https://github.com/packit/dashboard/pull/203))
```

Reply:
sorry to ruin the party, but I don't think users are interested in such info